plugins/analytics-module-ga/config.d.ts

```diff
@@ -27,6 +27,13 @@ export interface Config {
          */
         trackingId: string;
 
+        /**
+         * URL to Google Analytics analytics.js script
+         * Defaults to https://www.google-analytics.com/analytics.js
+         * @visibility frontend
+         */
+        gaAddress?: string;
```

Comment:
Maybe we should call this `scriptSrc` or smth? It's kind of already within the context of GA so don't really need that prefix I'm thinking

Reply:
sure!